legacy/application/common/CeleryManager.php

```diff
@@ -38,12 +42,11 @@ private static function _setupCeleryExchange($config, $exchange, $queue)
             $config['rabbitmq']['user'],
             $config['rabbitmq']['password'],
             $config['rabbitmq']['vhost'],
-            $exchange,                        // Exchange name
-                          $queue,                           // Binding/queue
-                          $config['rabbitmq']['port'],
+            $exchange,
+            $queue,
+            $config['rabbitmq']['port'],
             false,
-            true,                             // Persistent messages
```

Comment:
Was removing this parameter intentional?

Reply:
Yup: https://github.com/jooola/celery-php/blob/master/CHANGELOG.md